test/module/test_icp.py

```diff
@@ -32,31 +32,30 @@ def l_shape_wall(b, side1, side2, resolution=0.01, noise_std=0.01):
         pc2 = pc1 + noise
 
         tf = pp.randn_SE3(b)
-        print("The true tf is", tf)
         pc2 = tf.unsqueeze(-2).Act(pc2)
 
         return pc1, pc2, tf
 
     def random_point_cloud(b, num_points):
         pc1 = torch.rand(b, num_points, 3)
         tf = pp.randn_SE3(b)
-        print("The true tf is", tf)
         pc2 = tf.unsqueeze(-2).Act(pc1)
         return pc1, pc2, tf
```

Comment:
You could write the following test:
1. obtain a point cloud from somewhere
2. Keep two copies of the point cloud, randomly permute one of them and then apply random *known* transformations
3. Input the two point clouds to ICP, and use ICP to recover the transformation. Compare the result with the known transformation.